src/main.js

```diff
@@ -152,8 +155,40 @@ function savePoster(){
   savedPoster.classList.remove("hidden");
   mainPoster.classList.add("hidden");
 }
+
 function showMain(){
   mainPoster.classList.remove("hidden");
-  posterForm.classList.add("hidden");
```

Comment:
on line 184 we are creating the object instance, after the key word "new" calling a class the fist letter should be capitalized. 
this is what it should look like:
var newPoster = new Poster(newImageURL, newPosterTitle, newPosterQuote);

Reply:
Updated my code, will push again.